packages/cli/src/ui/hooks/usePhraseCycler.test.tsx

```diff
@@ -39,46 +39,77 @@ describe('usePhraseCycler', () => {
 
   it('should initialize with a witty phrase when not active and not waiting', () => {
     vi.spyOn(Math, 'random').mockImplementation(() => 0.5); // Always witty
-    const { lastFrame } = render(
-      <TestComponent isActive={false} isWaiting={false} />,
-    );
-    expect(WITTY_LOADING_PHRASES).toContain(lastFrame());
+    let lastFrame: () => string | undefined;
+    act(() => {
+      const result = render(
```

Comment:
Use our custom render, comment applies to whole file

Reply:
also reverted and re-fixed this file